themes/label-10.yaml

```diff
@@ -12,8 +12,8 @@ global:
     text_visible_island:                true
     icon_visible_landuse_green:         true     # needs work
     text_visible_landuse_green:         true
-    icon_size_green:                    [[13, '14px'], [16, '18px'], [18, '19px']]      # 0px for no, default, more  [[13, '14px'], [16, '18px'], [18, '19px']]
-    icon_size_green_l:                  [[14, '24px'], [16, '32px']]                    # 0px for no, default, more  [[14, '24px'], [16, '32px']]
+    icon_size_green:                    [[13, [95%,95%]], [16, [100%,100%]], [18, [105%,105%]]]
```

Comment:
This and line below can be simplified to have one % value for both dimensions (see similar comments).

Reply:
https://github.com/tangrams/bubble-wrap/commit/ce05472e6916d311fcef4071e6b1f7d53d629295